Public/weeblist.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>WeebList</title>
+    <link rel="stylesheet" href="weeblist.css" />
```

Comment:
Your linkage are wrong for css,Js. It should be like: (../assets/css/weeblist.css)